rhv_checks.py

```diff
@@ -338,6 +341,41 @@ def check_vms_distributed_hosts(system, warn=5, crit=10):
         sys.exit(3)
 
 
+def check_services_status(system, **kwargs):
+    """Check to see if service are in the desired state"""
+    hosts = system.api.system_service().hosts_service()
+    hosts_agents = dict()
+    hosts_status = dict()
+
+    for host in hosts.list():
+        host_service = hosts.host_service(host.id)
+        ssh = ssh_client(host_service, username="root", password=system.api._password)
+        with ssh:
+            for service_name, status in kwargs.iteritems():
+                service = is_service_in_status(ssh, service_name, status)
```

Comment:
@lina-nikiforova small thing, can you please rename this `service_status` or `service_current_status` instead of just `service`  ??

Reply:
Also, output currently looks like: 
```
.Overall status is {'env-rhv42-01.cfme2.lab.eng.rdu2.redhat.com': {u'ovirt-ha-agent': False, u'vdsmd': True}, 'env-rhv42-03.cfme2.lab.eng.rdu2.redhat.com': {u'ovirt-ha-agent': False, u'vdsmd': True}, 'env-rhv42-02.cfme2.lab.eng.rdu2.redhat.com': {u'ovirt-ha-agent': False, u'vdsmd': True}}
```
Can we replace `True` and `False` with maybe `Service in desired state` or `Service not in desired state` ? 

Just for readability. Thoughts?

Reply:
@kedark3 sure we can. But I'm not sure it increases the readability... If there're a lot of services (we have 5 already), those long messages are not easy to read... What do you think?

Reply:
@lina-nikiforova that's true as well. Maybe we can do this: 
```
("Critical: These hosts don't have all agents in the desired state: {}."
              "True: desired state, False:Not desired state"
               "Overall status is {}".format(trouble_hosts, hosts_agents))
```